app/javascript/components/common/CLIWalkthrough.tsx

```diff
@@ -9,15 +9,18 @@ declare global {
 }
 
 export default ({ html }: { html: string }): JSX.Element => {
+  const hasRun = useRef(false)
+  const scrollToTop = () => {
+    window.scrollTo({ top: 0 })
+  }
   useEffect(() => {
-    window.story = new Story($('tw-storydata'))
-    window.story.start($('#main'))
+    if (!hasRun.current) {
+      window.story = new Story($('tw-storydata'))
```

Comment:
Yeah - and as a short-term thing, we should ensure we're only loading it on the page its needed on etc.

Reply:
Yes, me neither. and this is a strange pattern, since rendering shouldn't be done by a useEffect. I didn't change this, as this has little to do with this PR, but I'd like to get rid of this dep too.